commands/commands.go

```diff
@@ -28,6 +27,36 @@ type command struct {
 	exec func(context.Context, *discordgo.Session, *discordgo.MessageCreate, []string) error
 }
 
+func init() {
+	commMap = map[string]*command{}
+
+	err := LoadFromStorage("./storage/aliases.json", &savedAliases)
```

Comment:
We take the address of savedAliases here, but then in that function body in json.Unmarshal, we take the address of it again? Is that intended behavior? Also worth saying that a map is a pointer type anyway I think.

Reply:
It's a mutation method so it modifies `savedAliases` when it returns. If you don't pass it as a pointer, it creates a copy of the variable inside the method instead of modifying the original one.